srcpkgs/snapcast/template

```diff
@@ -1,21 +1,23 @@
 # Template file for 'snapcast'
 pkgname=snapcast
-version=0.22.0
+version=0.23.0
 revision=1
 build_style=cmake
-configure_args="-DBUILD_WITH_TREMOR=OFF -DBUILD_WITH_AVAHI=$(vopt_if avahi ON OFF)
- -DCMAKE_INSTALL_SYSCONFDIR=/etc"
+configure_args="-DCMAKE_INSTALL_SYSCONFDIR=/etc -DBUILD_WITH_TREMOR=OFF
+ -DBUILD_WITH_AVAHI=$(vopt_if avahi ON OFF)
+ -DBUILD_WITH_PULSE=$(vopt_if pulseaudio ON OFF)"
 hostmakedepends="pkg-config"
 makedepends="alsa-lib-devel boost-devel expat-devel libflac-devel libsoxr-devel
- libvorbis-devel opus-devel $(vopt_if avahi avahi-libs-devel)"
+ libvorbis-devel opus-devel $(vopt_if avahi avahi-libs-devel)
+ $(vopt_if pulseaudio pulseaudio-devel)"
 short_desc="Synchronous multi-room audio player"
 maintainer="amak <amak.git@outlook.com>"
 license="GPL-3.0-or-later"
 homepage="https://github.com/badaix/snapcast"
 distfiles="https://github.com/badaix/snapcast/archive/v${version}.tar.gz"
-checksum=b5156f346c32557bc1347c81fd5071fd4a32be61adc582e63323c11b105c9ff6
+checksum=70efdeea60021f493f77ba1f3d00784911463cad11c6df214ecb19d74899b611
 
-build_options="avahi"
+build_options="avahi pulseaudio"
```

Comment:
Does enabling pulse disable alsa? I wonder whether the `pulseaudio` option ought to be enabled by default if it doesn't break anything. For that matter, if `avahi` enables client and server automatic discovery, it might be worth enabling by default.

Reply:
I've now enabled both options. Enabling `pulseaudio` shouldn't break `alsa`. The Snapcast client has a `player` option to configure which backend to use. The default is `alsa`.